src/runpython.c

```diff
@@ -68,3 +68,11 @@ runpython_init_py()
   Py_DECREF(d);
   return 0;
 }
+
+EM_JS(int, runpython_finalize_js, (), {
+  Module.version = function() {
+	Module.runPython("import pyodide");
+	return Module.runPython("pyodide.__version__");
+  };
```

Comment:
I tried here to set this as [a getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get#Defining_a_getter_on_existing_objects_using_defineProperty) with,
```js
Object.defineProperty(
   Module,
   'version', {
      get: function() { ... }
   });
```
however I was getting compilation errors; `EM_JS` doesn't seem to support it, unless I missed something.

The other alternatives could be to,
 - just do find&replace with `sed` similarly to how it is done in `src/pyodide.py` : the problem is that in the case when the version is not taken from
  git there is a chance that these two would not be consistent
 - calculate the property when `runpython_finalize_js` is called: not so keen on executing non critical code on interpreter initialization, but maybe the overhead is negligible
 - don't expose it in JS, and let users get it manually by accessing the value in python (from JS).